subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -222,3 +229,97 @@ void ull_conn_iso_done(struct node_rx_event_done *done)
 			  ((void *)conn == ull_disable_mark_get()));
 	}
 }
+
+static void ticker_resume_op_cb(uint32_t status, void *param)
+{
+	ARG_UNUSED(param);
+
+	LL_ASSERT(status == TICKER_STATUS_SUCCESS);
+}
+
+static void ticker_resume_cb(uint32_t ticks_at_expire, uint32_t remainder,
+			     uint16_t lazy, uint8_t force, void *param)
+{
+	static memq_link_t link;
+	static struct mayfly mfy = {0, 0, &link, NULL, lll_resume};
+	struct lll_event *resume_event;
+	struct lll_conn_iso_group *cig;
+	uint32_t ret;
+
+	LL_ASSERT(lazy == 0);
+
+	resume_event = param;
+	cig = resume_event->prepare_param.param;
+
+	/* Append timing parameters */
+	resume_event->prepare_param.ticks_at_expire = ticks_at_expire;
+	resume_event->prepare_param.remainder = remainder;
+	resume_event->prepare_param.lazy = 0;
+	resume_event->prepare_param.force = force;
+	resume_event->prepare_param.param = cig;
+	mfy.param = resume_event;
+
+	/* Kick LLL resume */
+	ret = mayfly_enqueue(TICKER_USER_ID_ULL_HIGH, TICKER_USER_ID_LLL,
+			     0, &mfy);
+
+	LL_ASSERT(!ret);
+}
+
+void ull_conn_iso_resume_ticker_start(struct lll_event *resume_event,
+				      uint8_t  resume_cis,
+				      uint32_t ticks_anchor,
+				      uint32_t resume_timeout)
+{
+	struct lll_conn_iso_group *cig;
+	uint32_t ready_delay_us;
+	uint32_t resume_delay_us;
+	int32_t resume_offset_us;
+	uint8_t ticker_id;
+	uint32_t ret;
+
+	cig = resume_event->prepare_param.param;
+	ticker_id = TICKER_ID_CONN_ISO_RESUME_BASE + cig->handle;
+
+	cig->resume_cis = resume_cis;
+
+	if (0) {
+#if defined(CONFIG_BT_CTLR_PHY)
+	} else {
+		struct ll_conn_iso_stream *cis;
+		struct ll_conn *acl;
+
+		cis = ll_conn_iso_stream_get(cig->cis_handles[resume_cis]);
+		acl = ll_conn_get(cis->lll.acl_handle);
+
+		ready_delay_us = lll_radio_rx_ready_delay_get(acl->lll.phy_rx, 1);
+#else
+	} else {
+		ready_delay_us = lll_radio_rx_ready_delay_get(0, 0);
+#endif /* CONFIG_BT_CTLR_PHY */
+	}
```

Comment:
Why use the if (0) construct? It is much more readable as:

```
#if defined(CONFIG_BT_CTLR_PHY)
￼	struct ll_conn_iso_stream *cis;
￼	struct ll_conn *acl;
￼
￼	cis = ll_conn_iso_stream_get(cig->cis_handles[resume_cis]);
￼	acl = ll_conn_get(cis->lll.acl_handle);
￼
￼	ready_delay_us = lll_radio_rx_ready_delay_get(acl->lll.phy_rx, 1);
#else
￼	ready_delay_us = lll_radio_rx_ready_delay_get(0, 0);
#endif /* CONFIG_BT_CTLR_PHY */
```

Reply:
Controller "boilerplate". @cvinayak, what is preferred?

Reply:
And also, there is a purpose of creating a scope for declaring local variables. With your suggeswtion, the variables will have to got to the top, with own #ifdef

Reply:
You can always create a scope, it's still more readable without the if/else

Reply:
> You can always create a scope, it's still more readable without the if/else

That's true. I'll consider it and take input from other members.